src/Umbraco.Web/Models/ImageCropDataSet.cs

```diff
@@ -76,7 +76,7 @@ public bool HasCrop(string alias)
 
         public bool HasImage()
         {
-            return string.IsNullOrEmpty(Src);
+            return ! string.IsNullOrEmpty(Src);
```

Comment:
It is just coding style. Sure it will work in its current form too :) Thanks for commit man!